app/assets/javascripts/host_edit.js

```diff
@@ -76,6 +76,7 @@ function computeResourceSelected(item){
         if ($('#compute_resource').find('.alert-danger').length > 0) $('#compute_resource_tab a').addClass('tab-error');
         update_capabilities($('#capabilities').val());
         tfm.initByteSpinner();
+        tfm.initCounterSpinner();
```

Comment:
yes, maybe import everything under `tfm.numFields` (so we can use e.g. `tfm.numField.initAll`, `tfm.numField.initByte` etc) - also hide away the confusing "spinner" name on the way.